bin/specextract

```diff
@@ -1412,8 +1460,8 @@ def get_region_filter(full_filename):
         region = full_filename
 
 
-    if not region or str(region) in ["NULL", "NONE", "None",""," "]:
-        raise IOError("Please specify a valid spatial region filter for {} or use FOV region files.".format(full_filename))
+    if not region or str(region) in [None, "NULL", "NONE", "None",""," "]:
```

Comment:
As you have this logic in several places I would suggest creating a function like

```
# note: not checked it matches the logic exactly
def check_filename_is_set(filename):
    if not filename:
        return False
    filename = filename.strip().upper()
    if filename in ["", "NULL", "NONE"]:
        return False
    return True
```

for use here.